pkg/hintrunner/zero/zerohint_usort.go

```diff
@@ -26,3 +30,52 @@ func newUsortEnterScopeHint() hinter.Hinter {
 func createUsortEnterScopeHinter() (hinter.Hinter, error) {
 	return newUsortEnterScopeHint(), nil
 }
+
+func newUsortVerifyHinter(value hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "UsortVerify",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			positionsDictInterface, err := ctx.ScopeManager.GetVariableValue("positions_dict")
+
+			if err != nil {
+				return err
+			}
+
+			positionsDict, ok := positionsDictInterface.(map[uint64][]*fp.Element)
```

Comment:
I belive if you check [this](https://github.com/starkware-libs/cairo-lang/blob/efa9648f57568aad8f8a13fbf027d2de7c63c2c0/src/starkware/cairo/common/usort.cairo#L8) function, you will see, that verify usort is being called at the end and is being executed in the same scope as this [PR](https://github.com/NethermindEth/cairo-vm-go/pull/352/files) (implements usort method). Notice that positionsDict is of type map[fp.Element][]uint64. Make sure that all the types are compatible with this PR. 
I noticed, that in my [PR](https://github.com/NethermindEth/cairo-vm-go/pull/352/files) I need to pass positionsDict into the scope for you to read it in this hint, but I don't know if we can pass maps into scopes or is it the functionality of a DictManager. @rodrigo-pino do you have any thoughts?